<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScientistInfoDialog</class>
 <widget class="QDialog" name="ScientistInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info about scientist</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Year of birth:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Year of death:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Information</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLineEdit" name="info_display_name">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="info_display_gender">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="info_display_year_of_birth">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="info_display_year_of_death">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="info_scientist_info">
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Related computers</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTableWidget" name="info_related_computers">
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>350</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>28</number>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>info_display_name</tabstop>
  <tabstop>info_display_gender</tabstop>
  <tabstop>info_display_year_of_birth</tabstop>
  <tabstop>info_display_year_of_death</tabstop>
  <tabstop>info_scientist_info</tabstop>
  <tabstop>info_related_computers</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
